### ISSUE TITLE
Table '<schema>.schemaversions' doesn't exist

### ISSUE THREAD

**Issue by soenneker** [DbUp/DbUp#188]
As of dbup and dbup-mysql 3.3.4 I'm getting the exception:

`"Table '<schema>.schemaversions' doesn't exist"`

and the script runner fails.

This occurs when I drop the schema and create a new one. I've downloaded and tried using the latest source from master and still get the same exception. With version 3.3.3 or 3.3.2 it's successful.

**Comment by simoncuce** [comment]
Hope this helps....I had the same issue, but there is a work around. Unless you explicitly set the schema name, the VerfiyTableExistsCommand method will run a query that looks for all schemaversion tables across the whole db, rather than just the schema you are using. As a result, it will fail when creating a new table. The answer is to explicitly add the schema to the MySqlDatabase extension method.Se code below

```
 var upgrader =
                DeployChanges.To
                    .MySqlDatabase(connectionString, **schema)**
                    .WithScriptsFromFileSystem(path)
                    .WithTransactionPerScript()
                    .LogToConsole()
                    .Build();
```

**Comment by simoncuce** [comment]
It would seem that the nuget package 3.3.4 is a little behind the code here. Thus the overloader MySqlDatabase(connectionString, "test1") isn't in the nuget package. Just use the code here

**Comment by droyad** [comment]
PR: #195

**Comment by droyad** [comment]
I'm trying to determine whether this is fixed in `4.0.0`, but I couldn't reproduce it on `3.3.5` or the latest.  I do lack deep MySql knowledge

My Code:
```
var connectionString = "server=127.0.0.1;uid=robert;pwd=taMMedPJ1!;database=test2";

var upgrader =
	DeployChanges.To
		.MySqlDatabase(connectionString)
		.WithScript("Foo", @" DELIMITER $$
CREATE PROCEDURE new_proc4 (IN new_param VARCHAR(256))
BEGIN
INSERT INTO tbl (cell) VALUES (new_param);
END$$
DELIMITER ;")
		.LogToConsole()
		.Build();

var result = upgrader.PerformUpgrade();

if (!result.Successful)
{
	Console.WriteLine(result.Error);
	return;
}
Console.WriteLine("Success!");
```

What I did:
- Create a schema `test`
- Run the above code (setting the right database in the connection string)
- Create a schema `test2`
- Run the above code (setting the right database in the connection string)

It creates the SchemaVersion table in both databases

**Comment by droyad** [comment]
Nevermind, late night brain fritz. This was fixed in `3.3.5`
